v22.1/take-and-restore-locality-aware-backups.md

```diff
@@ -117,29 +117,43 @@ To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`](re
 
 ## Create an incremental locality-aware backup
 
-If you backup to a destination already containing a [full backup](take-full-and-incremental-backups.html#full-backups), an [incremental backup](take-full-and-incremental-backups.html#incremental-backups) will be appended to the full backup in a subdirectory. For example:
+If you backup to a destination already containing a [full backup](take-full-and-incremental-backups.html#full-backups), an [incremental backup](take-full-and-incremental-backups.html#incremental-backups) will be appended to the full backup in a subdirectory. When you're taking an incremental backup, you must ensure that the incremental backup localities match the full backup localities otherwise you will receive an error. Alternatively, take another full backup with the matching localities before running the incremental backup. 
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO LATEST IN
-	  ('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+There is different syntax for taking an incremental backup depending on where you need to store the backups:
 
-{{site.data.alerts.callout_info}}
-When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
+- To append your incremental backup to the full backup in the [`incrementals` directory](take-full-and-incremental-backups.html#backup-collections):
 
-It is recommended that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
-{{site.data.alerts.end}}
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	BACKUP INTO LATEST IN
+		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
+	~~~
 
-And if you want to explicitly control where your incremental backups go, specify the subdirectory in the `BACKUP` statement:
+	{{site.data.alerts.callout_info}}
+	When [restoring from an incremental locality-aware backup](#restore-from-an-incremental-locality-aware-backup), you need to include **every** locality ever used, even if it was only used once.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-> BACKUP INTO {subdirectory} IN
-		('s3://us-east-bucket?COCKROACH_LOCALITY=default', 's3://us-west-bucket?COCKROACH_LOCALITY=region%3Dus-west');
-~~~
+	It is necessary that the same localities be included for every incremental backup in the series of backups; however, only the `default` locality is required.
```

Comment:
nit: think we can do without this sentence in this reshuffle: seems to be adequately conveyed just above in line 133, and might be confusing with "only the default is required" seeming to counter "the same localities be included". In reality, for a loc-aware incremental restore: both are required for the thing to succeed. "required" for `default` only means that the code does some input/param validation for this value. The restore will still fail, even if `default` is included to pass input validation, because the 101st incremental I ran in this series used `us-east` once and I forgot to include it here. Just a thought!

Reply:
I actually think (and this was probably past me who wrote this) the first sentence in this note block is referring to the restore: every locality ever used must be included in the restore statement. Then, the sentence referring to `default` is for actually taking the backup... that is the `default` is required. If that's the case, then it's poor note construction. I'm going to double check this and rearrange as appropriate. But, totally see your point.

Reply:
Sorted, thanks!

Reply:
thanks, looks good, nice work sorting!!